### PR TITLE
Fsi updates 111820b

### PR DESCRIPTION
**Pull request by jbranson000**
https://github.com/adobe/xdm/issues/1017

### CODE REVIEW

File: components/mixins/experience-event/experienceevent-card-actions.schema.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                "$ref": "https://ns.adobe.com/xdm/datatypes/data/measure",
+                "description": "Tracks when a card has been locked."
+              },
+              "xdm:cardOrderNew": {
+                "title": "Order New Card",
+                "$ref": "https://ns.adobe.com/xdm/datatypes/data/measure",
+                "description": "Tracks when a card has been requested."
+              },
+              "xdm:cardOrderType": {
+                "title": "Card Order Type",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:60]: type missing

File: components/mixins/experience-event/experienceevent-card-actions.schema.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                "enum": ["new", "lost", "stolen", "never received", "damaged"],
+                "description": "Tracks the type of card order associated with a card order event."
+              },
+              "xdm:cardType": {
+                "title": "Card Type",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:65]: type missing

File: components/datatypes/form-applications.schema.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          "title": "Form Application ID",
+          "type": "string",
+          "description": "Identifier used during an application process."
+        },
+        "xdm:formApplicationRequestedAmount": {
+          "title": "Form Application Requested Amount",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Tracks the monetary value of the application."
+        },
+        "xdm:formApplicationStatus": {
```
Comment (mjasrotia) [inline components/datatypes/form-applications.schema.json:48]: type missing

File: components/mixins/experience-event/experienceevent-card-actions.schema.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  "title": "Card Actions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Card action details such as card type as well as activation and lock status.",
+  "definitions": {
+    "card-actions": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:19]: type not required as its a definition

File: components/mixins/experience-event/experienceevent-card-application-process.schema.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  "title": "Card Application Process",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Card comparison details such as count, addition, removal, and view.",
+  "definitions": {
+    "card-application-process": {
+      "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-application-process.schema.json:19]: type not required as its a definition

File: components/mixins/experience-event/experienceevent-card-actions.schema.json (hunk 1 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-card-actions.schema.json:60]: resolved with next push

File: components/mixins/experience-event/experienceevent-card-actions.schema.json (hunk 2 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-card-actions.schema.json:65]: resolved with next push

File: components/mixins/experience-event/experienceevent-card-actions.schema.json (hunk 4 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-card-actions.schema.json:19]: resolved with next push

File: components/mixins/experience-event/experienceevent-card-application-process.schema.json (hunk 5 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-card-application-process.schema.json:19]: resolved with next push

File: components/datatypes/form-applications.schema.json (hunk 3 above)
Comment (jbranson000) [reply components/datatypes/form-applications.schema.json:48]: resolved with next push

File: components/mixins/experience-event/experienceevent-card-actions.schema.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Card action details such as card type as well as activation and lock status.",
+  "definitions": {
+    "card-actions": {
+      "properties": {
+        "xdm:personalFinances": {
+          "properties": {
+            "xdm:cardActions": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:22]: type missing

File: components/mixins/experience-event/experienceevent-card-actions.schema.json [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": ["financial_services"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Card action details such as card type as well as activation and lock status.",
+  "definitions": {
+    "card-actions": {
+      "properties": {
+        "xdm:personalFinances": {
+          "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:21]: type missing

File: components/mixins/experience-event/experienceevent-card-actions.schema.json (hunk 6 above)
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:22]: Both type and properties missing

File: components/datatypes/comparisons.schema.json [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+        "xdm:comparisonImpressions": {
+          "title": "Comparison Impressions",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/impressions",
+          "description": "Extension of the impressions data type."
+        }
+      }
+    }
+  },
+  "meta:status": "experimental"
+}
```
Comment (mjasrotia) [inline components/datatypes/comparisons.schema.json:67]: If there is a definition, you need to have an AllOF section

File: components/datatypes/form-applications.schema.json [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+        "xdm:formApplicationUsage": {
+          "title": "Form Application Usage",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/toolusage",
+          "description": "Extension of toolUsage data type."
+        }
+      }
+    }
+  },
+  "meta:status": "experimental"
+}
```
Comment (mjasrotia) [inline components/datatypes/form-applications.schema.json:78]: If there is a definition, you need to have an AllOF section

File: components/datatypes/impressions.schema.json [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+        "xdm:impressionType": {
+          "title": "Impression Type",
+          "type": "string",
+          "description": "Tracks the type of impression in an impression array."
+        }
+      }
+    }
+  },
+  "meta:status": "experimental"
+}
```
Comment (mjasrotia) [inline components/datatypes/impressions.schema.json:49]: If there is a definition, you need to have an AllOF section

File: components/mixins/experience-event/experienceevent-card-actions.schema.json [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+                "description": "Tracks when a card has been unlocked."
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "meta:status": "experimental"
+}
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.schema.json:91]: If there is a definition, you need to have an AllOF section

File: components/datatypes/impressions.schema.json (hunk 10 above)
Comment (jbranson000) [reply components/datatypes/impressions.schema.json:49]: I'll update that globally

File: components/datatypes/file-transfer.schema.json [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+          "description": "Type of the file transfer, if applicable."
+        },
+        "xdm:fileType": {
+          "title": "File Type",
+          "type": "string",
+          "description": "Type or category of file being up/downloaded by the visitor."
+        },
+        "xdm:fileURL": {
+          "title": "File URL",
+          "type": "string",
```
Comment (mjasrotia) [inline components/datatypes/file-transfer.schema.json:67]: Can we put "format": "uri" here ?

File: components/datatypes/financial-account.schema.json [hunk 13]
```diff
@@ -33,15 +33,14 @@
...
-            "$ref": "https://ns.adobe.com/experience/campaign/experienceevent-profile-personal-details"
-          }
+            "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details"
+          },
+          "description": "Array of the owner(s) of the selected financial account. Extension of the personal profile details data type."
         },
         "xdm:financialAccountType": {
           "title": "Financial Account Type",
           "enum": ["checking", "savings", "retirement", "investment"],
-          "type": "string",
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:44]: type missing

File: components/datatypes/form-applications.schema.json [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+          "description": "Identifier used during an application process."
+        },
+        "xdm:formApplicationRequestedAmount": {
+          "title": "Form Application Requested Amount",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Tracks the monetary value of the application."
+        },
+        "xdm:formApplicationStatus": {
+          "title": "Form Application Status",
+          "enum": ["initiated", "in progress", "approved", "declined"],
```
Comment (mjasrotia) [inline components/datatypes/form-applications.schema.json:50]: type still missing

File: components/datatypes/file-transfer.schema.json (hunk 12 above)
Comment (jbranson000) [reply components/datatypes/file-transfer.schema.json:67]: Yes.  Added.

File: components/datatypes/form-applications.schema.json (hunk 14 above)
Comment (jbranson000) [reply components/datatypes/form-applications.schema.json:50]: Fixed

File: components/datatypes/form-applications.schema.json [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+          "description": "Captures the status of the application for any given point in the application process."
+        },
+        "xdm:formApplicationStatusUpdate": {
+          "title": "Form Application Status Update",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Captured when the status for an application changes, either from an online or offline update."
+        },
+        "xdm:FormApplicationType": {
```
Comment (mjasrotia) [inline components/datatypes/form-applications.schema.json:58]: "Form" should be "form"

Comment (jbranson000) [reply components/datatypes/form-applications.schema.json:59]: fixed

File: components/datatypes/financial-account.schema.json (hunk 13 above)
Comment (jbranson000) [reply components/datatypes/financial-account.schema.json:44]: fixed

**Comment by jbranson000** [comment]
closing per new meta tags and measure updates

**Pull request closed without merging**